repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/step/AbstractStep.java

```diff
@@ -267,7 +267,7 @@ protected void cleanMavenArtifacts() {
             }
         }
 
-        if (this.config.isClean()) {
+        if (this.inspector.getRepoLocalPath() != null && this.config.isClean()) {
```

Comment:
Actually the `getRepoLocalPath` is not null: if you look line 13 of `CloneRepository` it shows that we use this value.
However we could check the existence of `getRepoLocalPath() + ".git"`.

Reply:
Hmm, I see. Check out the new commit and tell me what you think, please.

Reply:
After I commit that I got doubt about something: does the check you told me is supposed to wrap the entire body of the method `cleanMavenArtifacts`? Because if yes, I need to change again.